### PR TITLE
s3ql: add deprecation date

### PR DESCRIPTION
**Pull request by fxcoudert**
In order to allow https://github.com/Homebrew/brew/pull/9478

### CODE REVIEW

File: Formula/s3ql.rb [hunk 1]
```diff
@@ -16,11 +16,10 @@ class S3ql < Formula
     sha256 "63b52252fa9acd84fe7af0812241ab35e72062044cfe0659163a39e47a76581d" => :high_sierra
   end
 
-  deprecate! because: "requires FUSE"
-
-  # disable due to osxfuse API is with fuse2
-  # logged an issue, https://github.com/s3ql/s3ql/issues/192
-  deprecate! because: :does_not_build
+  # Requires fuse3, while osxfusy only has fuse2 API
```
Comment (SMillerDev) [inline Formula/s3ql.rb:19]:
```suggestion
  # Requires fuse3, while osxfuse only has fuse2 API
```

Comment (fxcoudert) [reply Formula/s3ql.rb]: Need more ☕

Comment (fxcoudert) [reply Formula/s3ql.rb]: Or less ☕ , maybe

**Review by SMillerDev — APPROVED** [review]
Maybe you need coffee beer: https://www.beeradvocate.com/beer/profile/345/184932/

**Comment by fxcoudert** [comment]
> coffee beer

That can't be legal. Shouldn't.

**Pull request merged**